src/libraries/periphery/MorphoLib.sol

```diff
@@ -8,6 +8,8 @@ import {MorphoStorageLib} from "./MorphoStorageLib.sol";
 /// @author Morpho Labs
 /// @custom:contact security@morpho.org
 /// @notice Helper library to access Morpho storage variables.
+/// @dev Warning: Supply and borrow getters may returned outdated values that do not include accrued interest and fees
+/// for instance.
```

Comment:
```suggestion
/// @dev Warning: Supply and borrow getters may return outdated values that do not include accrued interest.
```

I think fees are a cut on the "raw" interest so I don't think they impact how outdated values are here: it's just the accrued interest (which we consider being raw interest minus fees)

Reply:
Actually in the comment we don't mention "raw interests", so it can work for both. Marking this a resolved then